xml/System.Xml.Linq/XElement.xml

```diff
@@ -3676,12 +3676,20 @@ Root                1    2
         <Parameter Name="cancellationToken" Type="System.Threading.CancellationToken" Index="2" FrameworkAlternate="netcore-2.0;netcore-2.1;netcore-2.2;netcore-3.0;netstandard-2.1" />
       </Parameters>
       <Docs>
-        <param name="stream">To be added.</param>
-        <param name="options">To be added.</param>
-        <param name="cancellationToken">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="stream">A <see cref="T:System.IO.Stream" /> containing the raw XML to read into the newly created <see cref="T:System.Xml.Linq.XElement" />.</param>
+        <param name="options">A set of <see cref="T:System.Xml.Linq.LoadOptions" />.</param>
+        <param name="cancellationToken">A cancellation token.</param>
+        <summary>Create a new <see cref="T:System.Xml.Linq.XElement" /> and initialize its underlying XML tree using the passed <see cref="T:System.IO.Stream" /> parameter.  Optionally whitespace handling can be preserved.</summary>
+        <returns>A new <see cref="T:System.Xml.Linq.XElement" /> containing the contents of the passed in <see cref="T:System.IO.Stream" />.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+If <xref:System.Xml.Linq.LoadOptions.PreserveWhitespace> is enabled then the <xref:System.Xml.XmlReaderSettings> property <xref:System.Xml.XmlReaderSettings.IgnoreWhitespace> is set to `false`.
```

Comment:
@mairaw I think this remark should be moved to either the PreserveWhitespace API or the IgnoreWhitespace API. It doesn't seem right to mention it inside the LoadAsync remarks. Would you agree?

Reply:
Perhaps. It could also be just poorly written. Perhaps they meant:

```suggestion
If `options` is set to <xref:System.Xml.Linq.LoadOptions.PreserveWhitespace>, then the <xref:System.Xml.XmlReaderSettings.IgnoreWhitespace> property is set to `false`.
```